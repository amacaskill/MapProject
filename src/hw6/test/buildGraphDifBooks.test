LoadGraph sameChars buildGraphDifBooks.tsv

ListNodes sameChars

ListChildren sameChars char1
ListChildren sameChars char2 
