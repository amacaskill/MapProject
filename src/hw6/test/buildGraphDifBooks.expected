loaded graph sameChars

sameChars contains: char1 char2

the children of char1 in sameChars are: char2(book1) char2(book2) char2(book3) char2(book4) char2(book5)
the children of char2 in sameChars are: char1(book1) char1(book2) char1(book3) char1(book4) char1(book5)
